_data/codebase/v2_3/open-source/bin-magento.yml

```diff
@@ -5107,7 +5107,11 @@ commands:
 
         <comment>bin/magento queue:consumers:start someConsumer --area-code='adminhtml'</comment>
 
-    To save PID enter path:
+    To do not run multiple copies of one consumer simultaneously:
```

Comment:
@BaDos  To clarify -- the command shown is running a consumer in a single thread?  So the text should be:

```To run a consumer in a single thread and prevent multiple copies from running simultaneously:  ```